README.md

```diff
@@ -25,21 +27,28 @@ The following example shows a typical topology for a Key Protect instance:
 │   ├── root-key-ocp-cluster-...
 ```
 
+## Using HPCS instead of Key Protect
+
+You can use this module to create key rings and keys in Hyper Protect Crypto Services (HPCS) instead of Key Protect.
+
+Provide the GUID of your HPCS instance in the `var.existing_key_protect_instance_guid` input variable, and set `var.create_key_protect_instance` to `false`. Key rings and keys that you specify in the key map are created in the HPCS instance instead of Key Protect.
+
 ## Multiple Key Protect instances, and potential future directions for this module
+
 The strings `cos-bucket` and `ocp-cluster` are the cluster IDs for Cloud Object Storage and for the OpenShift Container Platform.
 
-The module supports only a single Key Protect instance and creates the key topology in that instance. It doesn't create multiple Key Protect instances. However, in a typical production environment, services might need multiple Key Protect instances for compliance reasons.
+The module supports only a single KMS instance and creates the key topology in that instance. It doesn't create multiple Key Protect instances, or support key rings and keys across multiple HPCS instances. However, in a typical production environment, services might need multiple Key Protect or HPCS instances for compliance reasons.
 
 For example, you might need isolation between regulatory boundaries (for example, between FedRamp and everything else). Or you might be required to isolate keys that are used by a service's control plane from the data plane (for example, with IBM Cloud Databases (ICD) services).
 
-To achieve compliance, you can write logic to call the module multiple times to create multiple Key Protect instances.
+To achieve compliance, you can write logic to call the module multiple times to utilize multiple KMS instances.
 
-One emerging pattern is to create one Key Protect instance per VPC. All workloads in the VPC access the Key Protect instance through a VPE binding. This simple approach ensures network segmentation. A drawback is that this approach creates more Key Protect instances than necessary, in some case.
+One emerging pattern is to have one KMS instance per VPC. All workloads in the VPC access the KMS instance through a VPE binding. This simple approach ensures network segmentation. A drawback is that this approach creates more KMS instances than necessary, in some case.
```

Comment:
```suggestion
One emerging pattern is to use one KMS instance per VPC. All workloads in the VPC access the KMS instance through a VPE binding. This simple approach ensures network segmentation. A drawback is that this approach creates more KMS instances than necessary, in some case.
```

again, just trying to use words that are less generic and easier for non-native speakers to understand.